test/Microsoft.AspNet.Localization.Tests/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+    "dependencies": {
+        "Microsoft.AspNet.Localization": "1.0.0-*",
+        "Microsoft.AspNet.TestHost": "1.0.0-*",
+        "Microsoft.Framework.Localization": "1.0.0-*",
```

Comment:
Once you remove `services.AddLocaliation()` from all tests, you won't need this reference. Please remove it.